### PR TITLE
Add tasks for Asyncy/Storyscript

### PR DESCRIPTION
**Pull request by stevepeak**
> CC @judepereira

Asyncy, a proud member of the CNCF/LF, presents four tasks that students of GSoC may choose. We are excited to provide the upmost support and mentoring as we take pride in our community development 👍 ❤️ 

> Asyncy is a fully open sourced platform under the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,31 @@ CoreDNS is a fast and flexible DNS server. It has a focus of service discovery i
 -	Recommended Skills: Golang, Microsoft Azure
 -	Mentor(s): Yong Tang (@yongtang)
 
+### Add annotations to Storyscript
+
+> [Storyscript](https://github.com/storyscript), by [Asyncy](https://asyncy.com) an open-source microservice choreography platform, is a language for stringing microservices together on Kubernetes. Asyncy is a CNCF/LF member.
```
Comment (nikhita) [inline README.md]: @caniszczyk @idvoretskyi Can CNCF/LF members also submit projects for GSoC or is it only the projects listed here https://www.cncf.io/projects?

**Comment by stevepeak** [comment]
Thank you for looking into it. I believe we have been asked to add Asyncy to the CNCF landscape, and will be doing so in about 2 months. If projects and landscape are one and the same than Asyncy is a project, just not listed yet.

File: README.md (hunk 1 above)
Comment (caniszczyk) [reply README.md]: it's only for CNCF projects

**Comment by caniszczyk** [comment]
GSOC is only meant for CNCF projects

**Comment by stevepeak** [comment]
@caniszczyk I understand, thank for the consideration. And just to I understand clearly: `CNCF Projects != CNCF Landscape` (https://landscape.cncf.io/) -- Gotcha!

**Pull request closed without merging**